mne_bids/write.py

```diff
@@ -21,7 +21,7 @@
 from mne import Epochs, events_from_annotations
 from mne.io.constants import FIFF
 from mne.io.pick import channel_type
-from mne.io import BaseRaw
+from mne.io import BaseRaw, anonymize_info, _stamp_to_dt
```

Comment:
I would just copy this function if it's a small one. Better to avoid using MNE private API as much as possible.